.github/workflows/build.yml

```diff
@@ -17,8 +17,9 @@ on:
   merge_group:
 
 env:
-  DOTNET_VERSION: ${{ '7.0.x' }}
+  DOTNET_VERSION: ${{ '7.0.100' }}
```

Comment:
Ah, mentioned this on the tooling PR, we should open an issue over there to look at this later.